components/observations/uploader/uploadImage.ts

```diff
@@ -0,0 +1,76 @@
+import axios from 'axios';
```

Comment:
this code is copied over from `submitObservation`, but basically unchanged otherwise.